app/src/main/res/values/colors.xml

```diff
@@ -3,4 +3,5 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="blue">#0000ff</color>
```

Comment:
please use upper case for these values - just as the values above. Thanks

Reply:
I've made the changes you required, please review

Reply:
I meant the actual color value

Reply:
Changes made

Reply:
thanks